<launch>
	<group ns="192_168_10_242">
		<group ns="artags">
			<!-- Marker tracker -->
			<arg name="marker_size" default="10" />
			<arg name="max_new_marker_error" default="0.08" />
			<arg name="max_track_error" default="0.2" />
			<arg name="cam_image_topic" default="/192_168_10_242/camera1/image_raw" />

			<arg name="cam_info_topic" default="/192_168_10_242/camera1/camera_info" />	
			<arg name="output_frame" default="camera1" />

			<node ns="artag1" name="ar_track_alvar1" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
		</group>
                <group ns="artags">
                        <!-- Marker tracker -->
                        <arg name="marker_size" default="10" />
                        <arg name="max_new_marker_error" default="0.08" />
                        <arg name="max_track_error" default="0.2" />
                        <arg name="cam_image_topic" default="/192_168_10_242/camera2/image_raw" />

                        <arg name="cam_info_topic" default="/192_168_10_242/camera2/camera_info" />
                        <arg name="output_frame" default="camera2" />

                        <node ns="artag2" name="ar_track_alvar2" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
		</group>

	</group>
</launch>
